arangod/RocksDBEngine/RocksDBBuilderIndex.cpp

```diff
@@ -23,11 +23,19 @@
 
 #include "RocksDBBuilderIndex.h"
 
+#undef USE_SST_INGESTION
```

Comment:
Readers of this code may wonder about the `#ifdef`, `#undef` here and try to find an according `#define`. As they will not find one, they may wonder if this code can be deleted.
Please leave a code comment here that explains the motivation for all of this, and that we want to keep the code around for later usage, once the issue is fixed in RocksDB that is keeping us from using SST file ingestion.

Reply:
Comment not addressed.

Reply:
done